src/util/cmdlineargs.cpp

```diff
@@ -111,6 +111,7 @@ bool CmdlineArgs::parse(const QStringList& arguments, CmdlineArgs::ParseMode mod
                 MIXXX_MANUAL_COMMANDLINEOPTIONS_URL);
     }
     // add options
+    // full-screen
```

Comment:
I really don't think these comments are necessary but they're here anyways so keep them

Reply:
well, they don't add much value, except (for me) their useful to see a new section begins:
//comment "marker" vs. skim through first line after a break to figure if it's about a new arg, or not
so basically just a small time-saver

Reply:
Sure. As I said, you already did the work, so lets keep em.